AMW_angular/io/src/app/deployments/deployments.component.ts

```diff
@@ -22,6 +22,13 @@ declare var $: any;
 @Component({
   selector: 'amw-deployments',
   templateUrl: './deployments.component.html',
+  styles: [`
```

Comment:
If you create a dedicated toast component, this should go into this component too.